builder/dockerfile/dispatchers.go

```diff
@@ -157,6 +158,7 @@ func add(req dispatchRequest) error {
 	if err != nil {
 		return err
 	}
+	copyInstruction.flChown = flChown.Value
```

Comment:
Are we sure we want to add this to `ADD` as well? We only have `--from` on `COPY`, and I think that's by design.

I thought `ADD` was kind of being deprecated/frozen.